starter-code/apple-pie-starter-code/index.html

```diff
@@ -2,34 +2,63 @@
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+  <link rel="stylesheet" type="text/css" href="css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,600" rel="stylesheet">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
+  <header class="header center">
+    <h1 class="header-text center">Apple Pie</h1>
+  </header>
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+  <section>
+    <div class="section">
+      <p class="summary-text center">This was my grandmother's apple pie recipe. I have never seen another one
+      quite like it. It will always be my favorite and has won me several first place
+      prizes in local competitions. I hope it becomes one of your favorites as well!
+      </p>
+      <img class="icon-image" src="images/recipe-info.png">
+    </div>
+  </section>
+  
+  <hr class="hr-style">
```

Comment:
You can use [border-bottom](https://www.w3schools.com/cssref/pr_border-bottom.asp) instead `hr`